test/e2e/specs/tools/advertising__promote.ts

```diff
@@ -35,7 +35,7 @@ skipDescribeIf( envVariables.ATOMIC_VARIATION === 'private' )(
 	DataHelper.createSuiteTitle( 'Advertising: Promote' ),
 	function () {
 		const pageTitle = DataHelper.getRandomPhrase();
-		const snippet = Array( 5 ).fill( DataHelper.getRandomPhrase() ).toString();
+		const snippet = Array( 2 ).fill( DataHelper.getRandomPhrase() ).toString();
```

Comment:
Repeating the same random phrase 5 times results in sometimes the character count exceeding what's allowed, resulting in the test suite failing.